src/status_im/signals/core.cljs

```diff
@@ -53,10 +55,27 @@
                 :peer-stats  peer-stats
                 :peers-count (count (:peers peer-stats)))}))
 
-(defn handle-local-pairing-signals
-  [event]
-  (log/debug "local pairing signal received"
-             {:event event}))
+(rf/defn handle-local-pairing-signals
+  [{:keys [db] :as cofx} event]
+  (log/info "local pairing signal received"
+            {:event event})
+  (let [connection-success?          (= (:type event) constants/local-pairing-event-connection-success)
+        completed-pairing?           (and (= (:type event) constants/local-pairing-event-process-success)
+                                          (= (:action event)
+                                             constants/local-pairing-action-pairing-account))
```

Comment:
there is no easy/clean way to know if pairing completed currently .. I'll make a separate PR for this @Samyoul